models/activities/action.go

```diff
@@ -253,6 +253,11 @@ func (a *Action) GetRepoName() string {
 	return a.Repo.Name
 }
 
+// GetRepoComposeMetas get the metas of the action repository.
+func (a *Action) GetRepoComposeMetas() map[string]string {
```

Comment:
Why it needs this function?

I guess it could be as simple as `$.Repo.ComposeMetas` in template withou this extra function?

Reply:
this is in action page for each user, no `$.Repo.ComposeMetas` is exist. and each ation may belong to different repo also.

Reply:
Sorry I didn't get the point.

If I understand correctly: the call `a.GetRepoComposeMetas()`  exactly equals to `a.Repo.ComposeMetas()`

So, it seems that every `.GetRepoComposeMetas` can be replaced by `.Repo.ComposeMetas` ?